components/typography/typography.css

```diff
@@ -0,0 +1,62 @@
+.heading1,
+.heading2,
+.heading3,
+.heading4,
+.heading5 {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+}
```

Comment:
You could have used a common class.

Reply:
I did it to avoid adding unnecessary classes